source/common/router/router.cc

```diff
@@ -637,6 +641,27 @@ Filter::streamResetReasonToResponseFlag(Http::StreamResetReason reset_reason) {
   throw std::invalid_argument("Unknown reset_reason");
 }
 
+const std::string Filter::resetReasonToString(Http::StreamResetReason reset_reason) const {
```

Comment:
Is this is the right place for this method? Should it be in an HTTP utility class potentially for use elsewhere?

Reply:
I was not sure if this is useful some where else - but does not harm moving to HTTP utility .